scripts/pr_review_test.py

```
@@ -778,15 +813,121 @@ def test_paths_dropped_by_the_token_budget_still_count_as_fetches(self) -> None:
+        ), mock.patch.object(pr_review, "call_model", return_value=decision):
+            verified, judged, _budget, _files, undecided = pr_review.judge_findings(
+                "owner/repo", "token", binding, "default", [candidate]
+            )
+        self.assertTrue(judged)
+        self.assertEqual(verified, [])
+        self.assertEqual(undecided, [candidate])
+
+    def test_cross_file_evidence_reads_consumers_and_tests_from_exact_head(self) -> None:
+        with tempfile.TemporaryDirectory() as directory:
+            root = pathlib.Path(directory)
+            subprocess.run(["git", "init", "-q", str(root)], check=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)